networking-calico/networking_calico/plugins/ml2/drivers/calico/test/test_plugin_etcd.py

```diff
@@ -892,6 +902,25 @@ def test_start_two_ports(self):
         self.assertEtcdWrites(expected_writes)
         self.assertEtcdDeletes(set())
 
+        # Change network used
+        _log.info("Change network used by endpoint HELLO")
+        context._port['network_id'] = 'calico-other-network-id'
+        self.osdb_ports[0]['network_id'] = 'calico-other-network-id'
+        self.driver.update_port_postcommit(context)
+
+        # Expected changes
+        ep_hello_value_v3['metadata']['labels'][
+            'projectcalico.org/openstack-network-name'] = 'my-first-network'
+        ep_hello_value_v3['metadata']['annotations'][
+            'openstack.projectcalico.org/network-id'] = \
+            'calico-other-network-id'
```

Comment:
Is this a leftover from an earlier idea?  I'm not sure why we are expecting this annotation.

Reply:
I'm changing the network-id on the neutron port, so I expect fields related to it on the endpoint (network-id, network-name) to change.

Reply:
Sorry, my mistake, I had forgotten about how this annotation works, but have reacquainted myself with that now.